src/runtime/const_noos_n64.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// TODO: Do not use GOTARGET tags in runtime. Come up with something smarter
+// based on some kind of memory description similar to the -M linker option.
+
+//go:build noos && n64
+
+package runtime
+
+const (
+	_OS                             = 0
+	noos                            = true
+	noosScaleDown                   = 2 // must be power of 2
+	noosStackCacheSize              = 16 * 1024
+	noosNumStackOrders              = 3
+	noosHeapAddrBits                = 23 // enough for 8 MiB K210 SRAM
+	noosLogHeapArenaBytes           = 17 // 128 KiB
+	noosArenaBaseOffset             = 0x0
```

Comment:
Does TLB is configured to start RAM at 0x0?

Reply:
It does